package.json

```diff
@@ -45,6 +45,7 @@
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
+    "clone": "^2.1.2",
```

Comment:
Does not seems to be used, since you went for `JSON.parse(JSON.stringify))` instead

Reply:
True! Wanted to ask you if that is good enough, or you want to pull in another dependency. Actually this change we should probably cherry-pick into master, because it gives rise to a nasty bug where when you zoom in and then load another dataset the "default zoom level" is set to the zoom of the previous dataset.

Reply:
Well, I am a bit ambivalent on very small JS dependencies where you end up depending on 10k packages all doing one small thing. Here, I think the JSON solution is pretty good, especially if it is commented.

Reply:
ok then perfect. just do it. if you think you can work on this PR and merge it before the end of the week, then I'd leave this in here, otherwise I think a bugfix applied on master would be a good idea as this change in plot range persisting across loading new datasets is weird.